CMakeLists.txt

```diff
@@ -5,8 +5,7 @@ include("common.cmake")
 # Haywire
 # ----------------------------------------
 project(haywire C)
-set(CMAKE_BUILD_TYPE RelWithDebInfo)
-#set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_BUILD_TYPE Release)
```

Comment:
Is there a reason why we don't want debug information? I don't really know a lot about these flags. This could be a great idea, please educate me :)